src/include/grpc_transcoding/internal/protobuf_types.h

```diff
@@ -15,13 +15,19 @@
 #ifndef GRPC_TRANSCODING_INTERNAL_PROTOBUF_TYPES_H_
 #define GRPC_TRANSCODING_INTERNAL_PROTOBUF_TYPES_H_
 
+#include "absl/strings/string_view.h"
+
 namespace google {
 namespace grpc {
 namespace transcoding {
 namespace internal {
 
 typedef ::google::protobuf::StringPiece string_view;
 
+inline absl::string_view ToAbslStringView(const string_view& s) {
```

Comment:
Can some of internal::string_view be replaced with absl::string_view since we have absl package now?  Ideally we like to replace all of them,  but I understand some of protobuf code still have to use its StringPiece.

Reply:
good catch, actually I found I don't even need this, it is only used in test private function.